# --- T2-COPYRIGHT-NOTE-BEGIN ---
# T2 SDE: package/*/wasi-libc/hotfix-no-bulk-memory.patch
# Copyright (C) 2022 The T2 SDE Project
# 
# This Copyright note is generated by scripts/Create-CopyPatch,
# more information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License version 2 as used by the T2 SDE.
# --- T2-COPYRIGHT-NOTE-END ---

--- wasi-libc-wasi-sdk-17/Makefile.vanilla	2022-12-20 13:10:47.170166499 +0100
+++ wasi-libc-wasi-sdk-17/Makefile	2022-12-20 13:12:21.237161761 +0100
@@ -483,8 +483,8 @@
 
 # TODO: apply -mbulk-memory globally, once
 # https://github.com/llvm/llvm-project/issues/52618 is resolved
-$(BULK_MEMORY_OBJS): CFLAGS += \
-        -mbulk-memory
+#$(BULK_MEMORY_OBJS): CFLAGS += \
+        #-mbulk-memory
 
 $(BULK_MEMORY_OBJS): CFLAGS += \
         -DBULK_MEMORY_THRESHOLD=$(BULK_MEMORY_THRESHOLD)
